code/example.py

```diff
@@ -0,0 +1,35 @@
+"""
+An example Python file which creates random data and exports it to a location specified
+by way of a
```

Comment:
`to a location specified in a command line argument`.

Reply:
Thank you! I've updated to add this in.